admin_manual/configuration/database/db_conversion.rst

```diff
@@ -8,6 +8,49 @@ If you have an existing ownCloud installation which uses SQLite, and you want to
 
 .. note:: ownCloud Enterprise edition does not support SQLite.
 
+
+Preparation
+-----------
+
+Add to your ownCloud config.php
+
+::
+
+	'mysql.utf8mb4' => true,
+
+Add or adjust in your /etc/mysql/mariadb.conf.d/50-server.cnf 
+
+..note::
+
+		You can do the same for mysql by replacing mariadb.conf.d/50-server.cnf with mysql.conf.d/mysqld.cnf
+
+
+::
+
+	key_buffer_size         = 32M
+	table_cache            = 400
```

Comment:
Dont know how that looks when you render, but if possible try to have the equal signs underneath for a better optic

Reply:
I would rather let @settermjd handle the formatting

Reply:
From that what I see, there are tabs before the key and spaces before the equal sign.
This would render the key underneath but the value as you see not underneath
Proposing to fix that